lb/lb.go

```
@@ -93,26 +92,36 @@ func (lb *LoadBalancer) worker() {
 				}
-				req.responseChan <- &response
+				req.ResponseChan <- &response
 			}
 		}
 	}
 }
 
-type Backend struct {
+func (lb *LoadBalancer) Close() {
+	close(lb.requestChan)
+	lb.svr.Shutdown(context.Background())
```

golangci-lint (1.62.2)

Error return value of `lb.svr.Shutdown` is not checked

(errcheck)